cms/cms-field-type.md

```diff
@@ -0,0 +1,519 @@
+---
+title: CMS field types
+---
+
+[[toc]]
+
+## Introduction
+
+When you configure `fields` of a CMS record, you need to define the type of each field.
+
+```yml
+records:
+  blogpost:
+    record_type: blogpost
+      list:
+        fields:
+          - name: content
+            type: String # CMS field types
+            label: Content
+```
+
+### Field type =/= Data type
+
+Field type in Skygear CMS refers to the UI presentation of the field, not its data type in the database.
+
+For example, suppose there are two roles in your app: student and teacher. In the user table, there is a column named 'role' the stores the role of the user. 'role' in the database has a data type of **String**, however, in the CMS, you can set the its field type to **Dropdown** to limit the input to student or teacher.
+
+```yml
+records:
+  user:
+   record_type: user
+     list:
+       fields:
+          - name: role
+            type: Dropdown # CMS field types
+            label: Role
```

Comment:
Need adding options for Dropdown type

For example
```
        - name: role  
          type: Dropdown
          label: Role
          options:
            - label: Teacher
              value: teacher
            - label: Student
              value: student
```

Reply:
fixed.